test/extended/ldap_groups.sh

```diff
@@ -229,6 +229,13 @@ for (( i=0; i<${#schema[@]}; i++ )); do
     popd > /dev/null
 done
 
+# special test for RFC2307
+pushd ${BASETMPDIR}/rfc2307 > /dev/null
+echo -e "\tTEST: Sync groups from LDAP server, tolerating errors"
+oadm groups sync --sync-config=sync-config-tolerating.yaml --confirm
+compare_and_cleanup valid_all_ldap_sync.yaml		
```

Comment:
I hadn't looked closely at these tests... all sync jobs should result in the same group memberships as specified in valid_all_ldap_sync?